### PR TITLE
Add possibillity to install via package

### PR DESCRIPTION
**Pull request by rwaffen**

### CODE REVIEW

**Comment by jhoblitt** [comment]
REFERENCE.md needs to be updated.  I'm not sure if that should be part of a non-release CI check but it is...

**Comment by rwaffen** [comment]
Forgetting this every time 😅 will update

File: spec/acceptance/init_spec.rb [hunk 1]
```diff
@@ -144,4 +144,20 @@ class { helm_binary:
       it { is_expected.to be_linked_to '/opt/helm/3.5.4/bin/helm' }
     end
   end
+
+  context 'with packaging set to package' do
+    let(:manifest) do
+      <<-PP
+      class { helm_binary:
+        packageing => 'package',
```
Comment (jhoblitt) [inline spec/acceptance/init_spec.rb:152]: I suspect `packageing` is the source of the CI failure.

**Review by jhoblitt — CHANGES_REQUESTED** [review]

**Comment by rwaffen** [comment]
oh sry, totally forgot about this. will add it to my queue again 😅

File: spec/acceptance/init_spec.rb (hunk 1 above)
Comment (rwaffen) [reply spec/acceptance/init_spec.rb:152]: it says the class has no such parameter, but it has, its part of the patch 🤔

Comment (rwaffen) [reply spec/acceptance/init_spec.rb:152]: ah damn, its a typo m(

**Comment by rwaffen** [comment]
@jhoblitt the centos images seem to be somehow broken, i don't see how i should get this to work...

**Comment by jhoblitt** [comment]
> @jhoblitt the centos images seem to be somehow broken, i don't see how i should get this to work...

It is close enough to the end that I think this module can drop EL7 support.

**Review by jhoblitt — APPROVED** [review]
LGTM.

**Comment by jhoblitt** [comment]
I'm labeling this as a breaking change since it drops EL7 support.

**Comment by jhoblitt** [comment]
Released as https://github.com/lsst-it/puppet-helm_binary/actions/runs/9715603649

**Pull request merged**